config/test.exs

```diff
@@ -46,5 +46,26 @@ config :ptolemy,
         auto_renew: true,
         opts: []
       }
+    },
+    server3: %{
```

Comment:
Combine server3 with server2 since they are both kv engines using approle. AFAIK the engine specifics have yet to be tested prior to this PR

Reply:
That's a good suggestion. Thanks! Fixed in the next commit